ui/selectors/selectors.js

```diff
@@ -623,6 +628,25 @@ export function getSubjectMetadata(state) {
   return state.metamask.subjectMetadata;
 }
 
+///: BEGIN:ONLY_INCLUDE_IN(mmi)
+export function getIsCustodianSupportedChain(state) {
+  const selectedIdentity = getSelectedIdentity(state);
+  const accountType = getAccountType(state);
+  const provider = getProvider(state);
+
+  const supportedChains =
+    accountType === 'custody'
+      ? getCustodyAccountSupportedChains(state, selectedIdentity.address)
+      : null;
+
+  return supportedChains?.supportedChains
+    ? supportedChains.supportedChains.includes(
+        Number(provider.chainId).toString(),
+      )
+    : true;
+}
+///: END:ONLY_INCLUDE_IN
```

Comment:
I can see why this is here and not in the duck, but can we extract this to selectors/institutional?